services/parser-database/keywordMock.py

```diff
@@ -0,0 +1,13 @@
+import random
+import json
+
+
+def get_keywords(text):
+    splited_text = text.split()
+    keywords = {
+        "keywords": [
+            splited_text[random.randint(0, len(splited_text) - 1)],
+            splited_text[random.randint(0, len(splited_text) - 1)],
+        ]
+    }
+    return json.dumps(keywords)
```

Comment:
Maybe a brief comment to describe what this line does. I am not really sure what `json.dumps` actually does.